bind-mount.c

```diff
@@ -158,29 +162,29 @@ get_mountflags (int proc_fd,
   end_token = skip_token (token, FALSE);
   *end_token = 0;
 
-  do {
-    end_token = strchr (token, ',');
-    if (end_token != NULL)
-      *end_token = 0;
+  do
+    {
+      end_token = strchr (token, ',');
+      if (end_token != NULL)
+        *end_token = 0;
 
-    for (i = 0; flags_data[i].name != NULL; i++)
-      {
+      for (i = 0; flags_data[i].name != NULL; i++)
```

Comment:
This drops the brackets because the inner block is just one statement, and is 3 lines or less.
However, I don't like that it drops it in this case because the inner block has an inner block, which makes it unclear where an "else" clause would go if added.

I have been unable to get uncruftify to understand this distinction though...